src/cmd/run.js

```diff
@@ -1,26 +1,26 @@
 /* @flow */
 import { fs } from 'mz';
 
-import defaultBuildExtension from './build';
+import defaultBuildExtension from './build.js';
 import {
   showDesktopNotification as defaultDesktopNotifications,
-} from '../util/desktop-notifier';
-import * as defaultFirefoxApp from '../firefox';
+} from '../util/desktop-notifier.js';
+import * as defaultFirefoxApp from '../firefox/index.js';
 import {
   connectWithMaxRetries as defaultFirefoxClient,
-} from '../firefox/remote';
-import {createLogger} from '../util/logger';
-import defaultGetValidatedManifest from '../util/manifest';
-import {UsageError} from '../../src/errors';
```

Comment:
@willdurand I looked into making sure the tests are importing the module they cover from src/ instead of lib/. 

One reason for doing that was that the module in src/ have the flowtype information and I can remove a couple of suppress comments that I had to add in the previous version to workaround flow complaining about missing type info.

The other reason is that I was a bit concerned that unit tests may run on an obsolete version of the transpiled files while developing a change locally, which isn't great from a developer workflow perspective.

While doing that, I caught a mistake that was already here:
this import was wrongly importing from a relative path that goes out of the src modules root dir and then points back to a src module, which was then turned into importing it from lib/ because it was rewritten by the jscodeshift transformer (which was expecting '/src/' to be part of a module url only for the tests).

That wasn't caught by the test suites because the module was there in lib and so everything kind of worked just fine, but I would say "just because of pure luck".

I fixed that in the last update to this PR, but I'm thinking that it would be nice if an eslint rule could error if any import is doing that by mistake in a new PR.